core/config/toml/types_test.go

```diff
@@ -143,7 +143,7 @@ func TestValidateConfig(t *testing.T) {
 				URL:                  &validSecretURL,
 				AllowSimplePasswords: &[]bool{true}[0],
 			},
-			skip:                !build.IsProd(),
```

Comment:
This would always be skipped since a `func Test` from a `_test.go` is only compiled in to test binaries. Instead, we test an unexported helper method that accepts a build mode parameter.